dietpi/dietpi-software

```diff
@@ -8550,8 +8550,7 @@ _EOF_
 			local ipv6=			
 			if grep -q '^[[:blank:]]*CONFIG_ENABLE_IPv6=1' /boot/dietpi.txt
 			then
-				G_WHIP_BUTTON_CANCEL_TEXT='Skip'
-				G_WHIP_INPUTBOX "Please enter your public IPv6 address. Skip this if you only want to use IPv4, or if you don't have a static IPv6 address." && ipv6=$G_WHIP_RETURNED_VALUE
+				ipv6=$(hostname -I | grep -o "[[:blank:]][[:digit:]:a-f]*")
```

Comment:
I think this cannot correctly be automated:
- It won't match if there is no IPv4 address assigned, rare but possible.
- We cannot know whether the first IPv6 address in this list is really the public address for the network interface that Tor clients connect through, or from any other network adapter, e.g. for a local network bridge or access point, or if there are multiple IPv6 addresses assigned to the same adapter, which is not uncommon.
- Ah and it includes a leading space, which is I think an issue in below `"ORPort [$ipv6]:$orport"`.

Probably the inputbox could stay, but with the derived IPv6 address as default value, e.g.:
```
for i in $(hostname -I)
do
	[[ i == *.* ]] && continue
	G_WHIP_DEFAULT_ITEM=$i
	break
done
```
If it contains a dot, it can only be an IPv4 address, else must be an IPv6 address. Probably there is an easier solution with grep/sed/mawk 😅.